nsot/conf/settings.py

```diff
@@ -45,13 +46,15 @@
 
 # The model to use to represent a User.
 AUTH_USER_MODEL = 'nsot.User'
+ANONYMOUS_USER_NAME = 'anonymous@service.local'
```

Comment:
Can you please move this to the `Guardian` section you created below and add a comment that explains why it's there?

Reply:
Done.